packages/app-mobile/components/get-responsive-value.test.ts

```diff
@@ -0,0 +1,265 @@
+			result = getResponsiveValue(input);
+			expect(result).toBe(correcOutput);
+		}
+	});
+
+	test('Should return correct value when only sm value is specified (Array Syntax)', () => {
+		const correcOutput = 40;
+		const input = [40];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
+			result = getResponsiveValue(input);
+			expect(result).toBe(correcOutput);
+		}
+	});
+
+	test('Should return correct value when sm and md values are specified (Object Syntax)', () => {
+		const input = { sm: 40, md: 70 };
+		const correctOutputs = [40, 40, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
+			result = getResponsiveValue(input);
+			expect(result).toBe(correctOutputs[i]);
+		}
+	});
+
+	test('Should return correct value when sm and md values are specified (Array Syntax)', () => {
+		const input = [40, 70];
+		const correctOutputs = [40, 40, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
+			result = getResponsiveValue(input);
+			expect(result).toBe(correctOutputs[i]);
+		}
+	});
+
+	test('Should return correct value when sm, md and lg values are specified (Object Syntax)', () => {
+		const input = { sm: 40, md: 70, lg: 90 };
+		const correctOutputs = [40, 40, 70, 70, 70, 90, 90, 90, 90, 90, 90, 90, 90];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
+			result = getResponsiveValue(input);
+			expect(result).toBe(correctOutputs[i]);
+		}
+	});
+
+	test('Should return correct value when sm, md and lg values are specified (Array Syntax)', () => {
+		const input = [40, 70, 90];
+		const correctOutputs = [40, 40, 70, 70, 70, 90, 90, 90, 90, 90, 90, 90, 90];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
+			result = getResponsiveValue(input);
+			expect(result).toBe(correctOutputs[i]);
+		}
+	});
+
+	test('Should return correct value when sm, md, lg and xl values are specified (Object Syntax)', () => {
+		const input = { sm: 40, md: 70, lg: 90, xl: 110 };
+		const correctOutputs = [40, 40, 70, 70, 70, 90, 90, 90, 110, 110, 110, 110, 110];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
+			result = getResponsiveValue(input);
+			expect(result).toBe(correctOutputs[i]);
+		}
+	});
+
+	test('Should return correct value when sm, md, lg and xl values are specified (Array Syntax)', () => {
+		const input = [40, 70, 90, 110];
+		const correctOutputs = [40, 40, 70, 70, 70, 90, 90, 90, 110, 110, 110, 110, 110];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
+			result = getResponsiveValue(input);
+			expect(result).toBe(correctOutputs[i]);
+		}
+	});
+
+	test('Should return correct value when sm, md, lg, xl and xxl values are specified (Object Syntax)', () => {
+		const input = { sm: 40, md: 70, lg: 90, xl: 110, xxl: 130 };
+		const correctOutputs = [40, 40, 70, 70, 70, 90, 90, 90, 110, 110, 110, 130, 130];
+		let result: number;
+
+		for (let i = 0; i < 13; i++) {
```

Comment:
Please don't hardcode values like this. That should be correctOutputs.length, shouldn't it?

Reply:
Yes, you're right. I'll fix that.